src/parse_stream.jl

```diff
@@ -55,7 +55,7 @@ function Base.summary(head::SyntaxHead)
 end
 
 function untokenize(head::SyntaxHead; unique=true, include_flag_suff=true)
-    str = untokenize(kind(head); unique=unique)
+    str = is_error(kind(head)) ? "error" : untokenize(kind(head); unique=unique)
```

Comment:
It seems a potentially dubious to special case this in here rather than in `untokenize(::Kind)`. I assume something fails somehow without this?

Reply:
Right now, the parser expects any error to `untokenize` to `"error"`. But now we have many error kinds. I could just add all of them to https://github.com/c42f/JuliaSyntax.jl/blob/77b4044218a7c7c5e34f8b1459a88fe6d405b64c/src/token_kinds.jl#L6 and have them sting to `"error"`.

Reply:
Yes I think they should be added to `_str_to_kind` — this allows them to be accessed with the `K` macro.